### PR TITLE
Feature/modify inspector realtime frontend

### PR DESCRIPTION
**Pull request by echoyang7**
- Add draggable split to inspector-realtime
- Add draggable split to inspector-advanced
- Support code editor to resize automatically
- Update inspector frontend code

### CODE REVIEW

File: frontend/src/store/inspector.js [hunk 1]
```diff
@@ -76,6 +83,64 @@ export default {
...
+              flow['_checked'] = true
+            }
+          }
+          commit('setCurrentFlowList', response.data)
+        })
+        .catch(error => {
+          bus.$emit('msg.error', 'Load flow list error: ' + error.data.message)
+        })
+    },
+    loadRecordMode ({ commit, state }) {
```
Comment (yumiguan) [inline frontend/src/store/inspector.js:101]: `state` is not used

File: frontend/src/store/inspector.js [hunk 2]
```diff
@@ -4,34 +4,41 @@ import { bus } from '@/eventbus'
...
     setActivitedGroup (state, group) {
       state.activatedGroup = group
     },
-    showDataButtons: function (state, val) {
-      state.showDataButtons = val
+    itemSeleted (state, val) {
+      state.itemSeleted = val
     },
-    search: function (state, val) {
+    search (state, val) {
```
Comment (yumiguan) [inline frontend/src/store/inspector.js:22]: search -> setSearchStr

File: frontend/src/views/inspector/ButtonBar.vue [hunk 3]
```diff
@@ -195,94 +186,30 @@ export default {
...
+      this.$store.dispatch('loadRecordMode')
     },
-    clearModalOk: function () {
-      this.$http.delete('/api/flow', { body: { ids: null } }).then(response => {
-      });
-
-      this.selectedFlow = null;
+    clearAllFlow () {
+      this.$store.dispatch('clearFlows')
+      this.$store.commit('clearSelectedId')
```
Comment (yumiguan) [inline frontend/src/views/inspector/ButtonBar.vue:199]: 可以在api成功清除了flow之后，再clearSelectedId

File: frontend/src/store/inspector.js [hunk 4]
```diff
@@ -4,34 +4,37 @@ import { bus } from '@/eventbus'
...
   },
   mutations: {
     setActivitedGroup (state, group) {
       state.activatedGroup = group
     },
-    showDataButtons: function (state, val) {
-      state.showDataButtons = val
-    },
-    search: function (state, val) {
+    setSearchstr (state, val) {
```
Comment (zhaoye) [inline frontend/src/store/inspector.js:18]: setSearchstr ==> setSearchStr

**Pull request merged**